### DISCUSSION TITLE
Mudraptor Gene and Exosuit rendering issue

### DISCUSSION THREAD

**Discussion by Sulphur22** [FakeFishGames/Barotrauma#14754] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

While having mudraptor genes and wearing an exosuit, the beak renders on top of the suit.

### Reproduction steps

1. Start a campaign
2. Splice mudraptor genes into someone
3. Get them to wear an exosuit

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly, fix coming in the update after the upcoming hotfix. Closing discussion.
